pyscf/adc/radc.py

```diff
@@ -194,11 +227,10 @@ class RADC(lib.StreamObject):
         'scf_energy', 'e_tot', 't1', 't2', 'frozen', 'chkfile',
         'max_space', 'mo_occ', 'max_cycle', 'imds', 'with_df', 'compute_properties',
         'approx_trans_moments', 'evec_print_tol', 'spec_factor_print_tol',
-        'E', 'U', 'P', 'X', 'ncvs', 'dip_mom', 'dip_mom_nuc'
+        'E', 'U', 'P', 'X', 'ncvs', 'dip_mom', 'dip_mom_nuc', 'if_heri_eris'
     }
 
-    def __init__(self, mf, frozen=0, mo_coeff=None, mo_occ=None):
-        from pyscf import gto
+    def __init__(self, mf, frozen=None, mo_coeff=None, mo_occ=None):
```

Comment:
Currently, none of the ADC modules print any information about the number of frozen and correlated orbitals in the output file. That's dangerous: the user will not know from the output file if their calculation used frozen core or not. I suggest that in the beginning of ADC calculations (ground- and excited-state) starting with verbose = 4 (logger.info) we print the number of frozen occupied, frozen virtual, active (correlated) occupied, and active (correlated) virtual orbitals. If frozen is a list, then we should print the contents of the list as well.